From 9817702434b1e15d2a885bf9601bb60b19b3b966 Mon Sep 17 00:00:00 2001
From: Jean Guyomarc'h <jean.guyomarch@openwide.fr>
Date: Mon, 18 Apr 2016 18:03:28 +0200
Subject: [PATCH] ARM: mach-imx mach-imx6q: fix spelling mistake in error
 message

The compatible device tree node that is searched for is imx6q-iomuxc-gpr
but was misspelled imx6q-iomux-gpr in the error handling message.

Signed-off-by: Jean Guyomarc'h <jean.guyomarch@openwide.fr>

diff --git a/arch/arm/mach-imx/mach-imx6q.c b/arch/arm/mach-imx/mach-imx6q.c
index 3878494b..0d83b84 100644
--- a/arch/arm/mach-imx/mach-imx6q.c
+++ b/arch/arm/mach-imx/mach-imx6q.c
@@ -220,7 +220,7 @@ static void __init imx6q_1588_init(void)
 				IMX6Q_GPR1_ENET_CLK_SEL_MASK,
 				clksel);
 	else
-		pr_err("failed to find fsl,imx6q-iomux-gpr regmap\n");
+		pr_err("failed to find fsl,imx6q-iomuxc-gpr regmap\n");
 
 	clk_put(enet_ref);
 put_ptp_clk:
-- 
2.1.4

